docs/dashboards/cloud-posture.asciidoc

```diff
@@ -1,2 +1,14 @@
 [[cloud-posture-dashboard]]
 = Cloud Posture Dashboard
+
+The Cloud Posture dashboard summarizes how your Kubernetes configuration measures up to security benchmarks. 
+
+NOTE: To learn how to collect this data, refer to <<kspm, Kubernetes security posture management>>.
+
+image::images/cloud-sec-dashboard.png[The Cloud Security dashboard]
+
+The first row of cards (Cloud Posture Score, Failed Findings, and Open Cases) summarizes your overall cloud security posture by aggregating data from all monitored Kubernetes clusters. Each subsequent row summarizes the posture of an individual Kubernetes cluster.
+
+The Cloud Posture Score card shows the percentage of your findings that passed over time. Hover your mouse to see precisely when the data was collected.
```

Comment:
I think hovering implies a mouse, so no need to mention it. Also revise to be less vision-oriented:
```suggestion
The Cloud Posture Score card shows the percentage of your findings that passed over time. Hover over the card to display when the data was collected.
```